src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/events/AppointmentEventPublisherTest.kt

```diff
@@ -66,4 +68,28 @@ class AppointmentEventPublisherTest {
     Assertions.assertThat(event.detailUrl).isEqualTo("http://localhost/action-plan/123/appointments/1")
     Assertions.assertThat(event.notifyPP).isTrue
   }
+
+  @Test
+  fun `builds an appointment session feedback event and publishes it`() {
+    val actionPlan = SampleData.sampleActionPlan()
```

Comment:
Not blocking at all, but I think we were trying to move away from using SampleData and towards using the Factory helpers

Reply:
Thanks for the reminder